<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonEth">
    <property name="geometry">
     <rect>
      <x>1811</x>
      <y>981</y>
      <width>39</width>
      <height>39</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/eth.png);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoLogin">
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>602</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>自动登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>908</x>
      <y>284</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRegist">
    <property name="geometry">
     <rect>
      <x>1065</x>
      <y>602</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonForgetPasswd">
    <property name="geometry">
     <rect>
      <x>1065</x>
      <y>570</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="whiteBlockWidget_1" native="true">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>430</y>
      <width>384</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 27px;</string>
    </property>
    <widget class="QLineEdit" name="lineEditId">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>321</width>
       <height>54</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>云针ID/手机号</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="whiteBlockWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>500</y>
      <width>384</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 27px;</string>
    </property>
    <widget class="QLineEdit" name="lineEditPasswd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>321</width>
       <height>54</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码                                                               6-16位</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxRemeberPasswd">
    <property name="geometry">
     <rect>
      <x>817</x>
      <y>570</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border:none;
</string>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLogin">
    <property name="geometry">
     <rect>
      <x>928</x>
      <y>686</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
image: url(:/images/login.png);
border-radius: 32px groove rgba(0,0, 0,0);
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>888</x>
      <y>644</y>
      <width>144</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(246, 76, 76);</string>
    </property>
    <property name="text">
     <string>请输入云针ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditPasswd</tabstop>
  <tabstop>lineEditId</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditPasswd</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonLogin</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>964</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>960</x>
     <y>726</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
